assets/docs/configuration/targets/http-full-example.hcl

```diff
@@ -17,6 +17,9 @@ target {
     # Request timeout in seconds (default: 5)
     request_timeout_in_seconds = 2
 
+    # Request timeout in milliseconds (default: 5000)
```

Comment:
Sorry for an extra nit - can we add a note here to say that this one takes precedence? This is what will show up in the docs!